sqlflow_models/deep_embedding_cluster.py

```diff
@@ -200,10 +224,15 @@ def sqlflow_train_loop(self, x, epochs=1, verbose=0):
         features = ite.get_next()
         self.predict(x=features)
 
+        # Get train.batch_size from sqlflow
```

Comment:
Maybe we can use `self._train_batch_size = tf.shape(features)[0]` instead?

Reply:
I have tried, but it seems not working.